openshift/masu.env.example

```diff
@@ -2,7 +2,9 @@ NAME=masu
 NAMESPACE=myproject
 BUILD_MEMORY_LIMIT=1Gi
 DATABASE_SERVICE_NAME=koku-pgsql
-MASU_CPU_LIMIT=1
+MASU_CPU_REQUEST='500m'
```

Comment:
seems to be inconsistent as to whether these are quoted.

Reply:
Removed the quotes. I copied the values from the yaml but I was able to successfully deploy without any quotes in the param-file